<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txtNumeros">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>111</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Numeros aleatorios</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtCola">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>111</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cola Circular</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pila</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtPila">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>111</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Lista Doble</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtLista">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>121</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Vector</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtVector">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>90</y>
      <width>111</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInsercion">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>190</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Inserción</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIntercambio">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>90</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Intercambio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSeleccion">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>140</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Selección</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBurbuja">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>240</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Burbuja</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShell">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>290</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Shell</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMergesort">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>340</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mergesort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuicksort">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>390</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quicksort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBinsort">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>440</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Binsort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRadixsort">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>490</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Radixsort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDesordenar">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>550</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Desordenar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ordenamiento</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIntercambio">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSeleccion">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInsercion">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>190</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBurbuja">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblShell">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>290</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMergsort">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblQuicksort">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>390</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBinsort">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRadixsort">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>490</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1063</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
